resource/filter.go

```diff
@@ -27,13 +27,19 @@ type TypeFilter struct {
 	Tags map[string]string `yaml:",omitempty"`
 	// select resources by creation time
 	Created *Created `yaml:",omitempty"`
+	Age     *Age     `yaml:",omitempty"`
 }
 
 type Created struct {
 	Before *time.Time `yaml:",omitempty"`
 	After  *time.Time `yaml:",omitempty"`
 }
 
+type Age struct {
+	Before *time.Duration `yaml:",omitempty"`
+	After  *time.Duration `yaml:",omitempty"`
```

Comment:
Not sure if these terms make sense or not. Any suggestions?
Maybe `younger/older`? or `earlier/later`?